app/renderer/js/components/webview.js

```diff
@@ -12,6 +12,15 @@ const { shell, app, dialog } = require('electron').remote;
 const BaseComponent = require(__dirname + '/../components/base.js');
 
 const shouldSilentWebview = ConfigUtil.getConfigItem('silent');
+
+// this hidden webview will be used to open pdf url and
+// save it to user's computer without triggering a reload
+// when navigating to pdf url to download it.
+const hiddenWebView = document.createElement('webview');
```

Comment:
@cPhost should we make a component for this like `hidden-webview.js` and init it?

Reply:
I will also put the function there which would make this more simpler.